.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
```

Comment:
It would be great to add `.serverless` folder to the ignored list

Reply:
Just realized that it wasn't there? Thanks!